Gemfile.lock

```diff
@@ -518,4 +522,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.1.4
+   2.2.22
```

Comment:
Please, do not bump bundler version at this time.

Reply:
Sorry this was done automatically by my tools, I'll manually undo this.